docs/how-to-guides/streams/manage-streams.md

```diff
@@ -0,0 +1,141 @@
+---
+id: manage-streams
+title: How to manage data streams with queries
```

Comment:
Dropdown is `Connect to stream sources`, then I would call this `Via cypher` or `Via LAB` in lab case. It makes more sense because of dropdown.  This is just suggestion about wording, feel free to ignore.

Reply:
Will this be added?